.github/workflows/cli.yml

```diff
@@ -89,18 +91,26 @@ jobs:
         with:
           go-version: ${{ env.golang-version }}
 
-      - name: Build
-        working-directory: cli
-        run: |
-          make build
-
       - name: Start kuberenetes cluster
         working-directory: cli
         env:
           KUBE_VERSION: ${{ matrix.kube-version }}
         run: |
           make start-kind
 
+      - name: Install cert-manager
+        working-directory: cli
+        run: |
+          kubectl apply -f https://github.com/cert-manager/cert-manager/releases/download/${{ env.cert-manager-version }}/cert-manager.yaml
+
+      - name: Build
+        working-directory: cli
+        run: |
+          make build
```

Comment:
This stage is executed after cluster creation, so kubernetes can do its job in the background while we focus on building tobs binary. In theory it should speed things up.